Glyssen/Bundle/GlyssenDblTextMetadata.cs

```diff
@@ -84,6 +85,9 @@ public override GlyssenDblMetadataLanguage Language
 		[XmlElement("projectStatus")]
 		public ProjectStatus ProjectStatus = new ProjectStatus();
 
+		[XmlElement("projectsettings")]
```

Comment:
I see...looks like attributes are all lowercase and elements are initial lowercase and then camel case.